### ISSUE TITLE
Improve the heading

### ISSUE THREAD

**Issue by sbrl** [ConnectedHumber/Air-Quality-Web#39]
Currently, the heading looks rather out-of-place - and it's been bugging me for a while. It would be great if it looked a bit more like the [main connected humber website](https://connectedhumber.org).

Changing it is easy - it's deciding what to change it _to_ that's the hard part. What should it look like?

/cc @BNNorman, @robinharris, @bsimmo

**Comment by BNNorman** [comment]
So, are you suggesting a blue header with map below? If so i agree. It would look more professional. I think it should include ' connected humber'. Should it also include 'air quality' or ' environment monitoring' or ' monitAIR'

**Comment by sbrl** [comment]
Yeah, something like that. Something more professional would be a good idea.

Indeed, rewording it would probably be a good idea too. Something short would be good I think.

Also, would _ConnectedHumber_ have a space in the middle or not?

While we're thinking about it, if anyone has the artistic skills to come up with a good logo I wouldn't be averse to setting a favourites icon (i.e. the icon next to the title on the tab at the top fo the screen). That could help people to identify it more quickly (e.g. in their history, bookmarks, etc.).

**Comment by BNNorman** [comment]
well since ConnectedHumber is not specifically tied to any tech/app I'd suggest keeping it simple. This is me playing about - maybe it'll inspire someone with more artitics flair. The blue was taken from the website. I'm leaning towards the cog...

![image](https://user-images.githubusercontent.com/15849181/60454697-69db8900-9c2c-11e9-974f-4c77ff383bbe.png)

**Comment by sbrl** [comment]
Oooh, nice! I like the cog too. I'd probably suggest using the same font as the main website heading though, to keep it consistent maybe?

**Comment by BNNorman** [comment]
Good point about the font. So I've eliminated the italic, changed to Tahoma and fiddled.

![image](https://user-images.githubusercontent.com/15849181/60487611-df307380-9c97-11e9-89dd-993740141e6a.png)

**Comment by BNNorman** [comment]
Vartiations on a cog - again ignore underlining.

![image](https://user-images.githubusercontent.com/15849181/60490398-5cf77d80-9c9e-11e9-93c7-9e32c5557e2f.png)

**Comment by sbrl** [comment]
Nice! I like the one on the bottom row 3rd one from the left the best personally, but the one on the far left on the same row might look better at small sizes. Some testing is required.

Note that for the icon a larger native size is actually better (e.g. 256x256), since the `apple-touch-icon` needs to be the larger size.

**Comment by BNNorman** [comment]
These are vector graphics so can be exported at any sizes you want. Ill export at 256,64 and 32 and attach them here in a few minutes.

**Comment by BNNorman** [comment]
Saved as svg images - had to zip them to attach here.

[CH_Logos.zip](https://github.com/ConnectedHumber/Air-Quality-Web/files/3356598/CH_Logos.zip)

**Comment by sbrl** [comment]
Nice, thanks! I'll add it to the web interface soon.

**Comment by sbrl** [comment]
Done :D

Now it's just the heading to do next.
